App/components/modals/NotificationModal.tsx

```diff
@@ -2,12 +2,13 @@ import { useNavigation } from '@react-navigation/core'
 import { StackNavigationProp } from '@react-navigation/stack'
 import React, { useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { Modal, StyleSheet, TouchableOpacity, View } from 'react-native'
+import { Modal, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
```

Comment:
We should just use the custom `Text` component since the colour theme is already referenced in it. Unless the text theme style cannot be applied to it as well.